src/S3/Exception/S3MultipartUploadException.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace Aws\S3\Exception;
+
+use Aws\CommandInterface;
+use Aws\Exception\AwsException;
+use Aws\Multipart\UploadState;
+
+class S3MultipartUploadException extends \Aws\Exception\MultipartUploadException
+{
+    /** @var string File path of transfer object */
+    private $filePath;
+
+    /**
+     * @param UploadState      $state Upload state at time of the exception.
+     * @param \Exception|array $prev  Exception being thrown.
```

Comment:
This seems to be missing information about when and why an array would be provided.